### PR TITLE
[BF-67] FEAT: Update Header/Footer

### PR DESCRIPTION
**Pull request by rikarsen**

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -19,74 +25,58 @@ const socials = [
   { href: 'http://gg.com', icon: <GGIcon /> },
 ]
 
+const copyright = 'Copyright © 2023 with all rights reserved'
```
Comment (ArthurO2002) [inline src/components/Footer/Footer.tsx:28]: Use translation

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -19,74 +25,58 @@ const socials = [
...
-          <li className="mb-2.5 text-base">
-            <Link
-              href="youtube.com"
-              className="hover:underline"
-            >
-              Terms of Use
-            </Link>
-          </li>
+      <div className="order-4 space-y-5 divide-gray-thin xl:order-2 xl:divide-y">
+        <p className="text-base font-medium">Policies and Terms of use:</p>
```
Comment (ArthurO2002) [inline src/components/Footer/Footer.tsx:43]: Use translations logic

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -19,74 +25,58 @@ const socials = [
...
-            <Link
-              key={social.href}
-              href={social.href}
-              target="_blank"
-              className="mr-5 flex h-12 w-12 items-center justify-center rounded-full bg-white"
-            >
-              {social.icon}
-            </Link>
+      <div className="order-3 mb-10 space-y-5 divide-gray-thin border-b border-gray-thin pb-10 xl:order-3 xl:m-0 xl:divide-y xl:border-none xl:p-0">
+        <p className="text-base font-medium">Don’t miss out:</p>
```
Comment (ArthurO2002) [inline src/components/Footer/Footer.tsx:51]: All texts to translations

File: src/components/Icons/Wave.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import clsxMerge from '@lib/clsxm'
+
+export const Wave: FC<IconProps> = ({ fill = 'fill-gray-wave', className, ...rest }) => {
+  const style = clsxMerge(fill, className)
+
+  return (
+    <svg
+      // width="1440"
+      // height="213"
+      viewBox="0 0 1440 213"
```
Comment (ArthurO2002) [inline src/components/Icons/Wave.tsx:12]: remove comments

File: src/styles/globals.css [hunk 5]
```diff
@@ -4,6 +4,20 @@
...
+    @apply outline outline-2 outline-primary;
+  }
+}
+
+.wave {
+  background-size: 50% 100%;
+  bottom: -5%;
+  height: calc(12 * 1vh);
+  left: 0;
+  opacity: 1;
```
Comment (ArthurO2002) [inline src/styles/globals.css:19]: Add todo to change later

File: src/styles/globals.css [hunk 6]
```diff
@@ -4,6 +4,20 @@
...
+  position: absolute;
+  right: 0;
+  width: 100vw;
```
Comment (ArthurO2002) [inline src/styles/globals.css:22]: use %

File: src/ui-components/Button/Button.tsx [hunk 7]
```diff
@@ -4,7 +4,7 @@ import clsxMerge from '@lib/clsxm'
 
 const ButtonVariant = ['contained', 'outlined', 'text'] as const
 const ButtonColor = ['primary', 'secondary', 'gray'] as const
-const ButtonSize = ['xs', 'sm', 'bs', 'lg', 'fl', 'hg', 'app'] as const
+const ButtonSize = ['xs', 'sm', 'sm2', 'bs', 'lg', 'fl', 'hg', 'app'] as const
```
Comment (ArthurO2002) [inline src/ui-components/Button/Button.tsx:7]: Can we add different naming for sm2 size?

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: src/components/Icons/Wave.tsx (hunk 4 above)
Comment (rikarsen) [reply src/components/Icons/Wave.tsx:12]: done

File: src/styles/globals.css (hunk 5 above)
Comment (rikarsen) [reply src/styles/globals.css:19]: done

File: src/styles/globals.css (hunk 6 above)
Comment (rikarsen) [reply src/styles/globals.css:22]: done

File: src/ui-components/Button/Button.tsx (hunk 7 above)
Comment (rikarsen) [reply src/ui-components/Button/Button.tsx:7]: done

File: src/components/Footer/Footer.tsx (hunk 1 above)
Comment (rikarsen) [reply src/components/Footer/Footer.tsx:28]: done

File: src/styles/globals.css (hunk 6 above)
Comment (ArthurO2002) [reply src/styles/globals.css:23]: Not Done

**Review by Arthur02V — APPROVED** [review]

**Review by ArthurO2002 — APPROVED** [review]

**Review by ArthurO2002 — APPROVED** [review]

**Pull request merged**